### ISSUE TITLE
TypeBodyLengthRule should not count conditional directives

### ISSUE THREAD

**Issue by Sega-Zero** [realm/SwiftLint#3139]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

type_body_length rule counts lines with conditional directives as lines of code, which is not correct. Example:
```swift
class Test {
  func foo() {
    #if DEBUG
    print("test")
    #endif 
  }
}
```
How many lines are in foo function? swiftlint thinks there are 5 lines, but there's 3 lines of code.
This rule ignores comments and whitespaces already since they are not an actual lines of code (#369). So, why a conditional directives does count?
I believe it's an incorrect rule behaviour.

### Environment

* SwiftLint version (run `swiftlint version` to be sure)?
0.39.1
* Installation method used (Homebrew, CocoaPods, building from source, etc)?
CocoaPods

swiftlint ouptut for sample above:
```ruby
$ pbpaste | swiftlint lint --no-cache --use-stdin --enable-all-rules
Loading configuration from '.swiftlint.yml'
<nopath>:1:1: warning: Type Body Length Violation: Type body should span 3 lines or less excluding comments and whitespace: currently spans 4 lines (type_body_length)
```
using sample yml:
```yaml

type_body_length: 3

```

**Comment by Sega-Zero** [comment]
commenting. still reproducing this
